packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -2,6 +2,40 @@
 title: Okta API Products Release Notes
 ---
 
+## March
+
+### Monthly Release 2021.03.0
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [The SAML 2.0 Assertion grant flow is now Self-Service Early Access (EA)](#the-saml-2-0-assertion-grant-flow-is-now-self-service-early-access-ea) | March 3, 2021 |
+| [The Okta Org API is now GA in Preview](#the-okta-org-api-is-now-ga-in-preview) | March 3, 2021         |
+| [The /login/token/redirect endpoint now has a dedicated rate limit](#the-login-token-redirect-endpoint-now-has-a-dedicated-rate-limit) | March 3, 2021          |
+| [The new LDAP Interface authentication type is now GA in Preview](#the-new-ldap-interface-authentication-type-is-now-ga-in-preview) | March 3, 2021          |
+| [Bugs fixed in 2021.03.0](#bugs-fixed-in-2021-03-0) | March 3, 2021         |
+
+#### The SAML 2.0 Assertion grant flow is now Self-Service Early Access (EA)
```

Comment:
Should we add "in Preview" to this title (and bullet link above)? In the blurb and elsewhere where we mention release type we seem to include "in preview"